bpf/bpf_host.c

```diff
@@ -1702,6 +1702,7 @@ int cil_to_host(struct __ctx_buff *ctx)
 		 */
 		traced = true;
 
+		barrier_data(ctx);
```

Comment:
![스크린샷 2025-12-27 오후 2 31 15](https://github.com/user-attachments/assets/4d8ac980-916e-4829-818c-d0dbaaf3f083)

The `barrier_data` was invoked because the verifier reported `dereference of modified ctx ptr disallowed`.
Is there a better approach than this?